### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: proto/api.pb.go [hunk 1]
```diff
@@ -872,15 +871,15 @@ func init() {
 }
 func (m *ClientCmdID) Unmarshal(data []byte) error {
 	l := len(data)
-	index := 0
-	for index < l {
+	iNdEx := 0
```
Comment (petermattis) [inline proto/api.pb.go]: wtf?

**Comment by petermattis** [comment]
LGTM

File: proto/api.pb.go (hunk 1 above)
Comment (tamird) [inline proto/api.pb.go]: yeah. https://github.com/gogo/protobuf/issues/72

**Comment by bdarnell** [comment]
LGTM

**Comment by tamird** [comment]
This is blocked on https://github.com/gogo/protobuf/pull/76 currently.

**Pull request merged**